plugins/modules/packaging/os/homebrew.py

```diff
@@ -127,6 +127,10 @@
     state: present
     install_options: with-baz,enable-debug
 
+- community.general.homebrew:
```

Comment:
Can you please add a `name:`:
```suggestion
- name: ... put something descriptive here ...
  community.general.homebrew:
```

Reply:
definitely :) - I'll make a small change for that

Reply:
@felixfontein thanks for that input, I've pushed a small change on the branch that provides more descriptive info on the name property - let me know if the description makes sense :) :+1: